lib/cartopy/mpl/geoaxes.py

```diff
@@ -872,6 +874,20 @@ def gridlines(self, crs=None, draw_labels=False, **kwargs):
         * draw_labels
             Label gridlines like axis ticks, around the edge.
 
+        * xlocs
+            An iterable of gridline locations or a
+            :class:`matplotlib.ticker.Locator` instance which will be used to
+            determine the locations of the gridlines in the x-coordinate of
+            the given CRS. Defaults to None, which implies automatic locating
```

Comment:
I'm comfortable merging the PR with the current terminology, but would be happy if @ajdawson were to submit another PR addressing.

For my 2 cents, the term locating is as good as definition, but it may be that there is another term, or way of re-phrasing which is more clear.